test/src/main/kotlin/com/bridge/ouroboros/compose/test/EffectMatching.kt

```diff
@@ -40,11 +33,3 @@ class EffectMatching<E>(
         }
     }
 }
-
-fun <E> haveNoEvents() = object : Matcher<EffectResult<E>> {
```

Comment:
We have exactly two places in Bridge where this is used and can be replaced by `expectNoEvents()`